extensions/confluence-search/src/index.tsx

```diff
@@ -1,23 +1,23 @@
 import {
   ActionPanel,
   CopyToClipboardAction,
-  List,
-  OpenInBrowserAction,
-  ImageMask,
   getPreferenceValues,
   Icon,
+  ImageMask,
+  List,
+  OpenInBrowserAction,
   showToast,
   ToastStyle,
 } from "@raycast/api";
-import { useState, useEffect, useRef, useCallback } from "react";
 import fetch, { AbortError, RequestInit, Response } from "node-fetch";
+import { useCallback, useEffect, useRef, useState } from "react";
 
 const prefs: { instance: string; user: string; token: string } = getPreferenceValues();
 export const confluenceUrl = `https://${prefs.instance}`;
 
 const headers = {
   Accept: "application/json",
-  Authorization: "Basic " + Buffer.from(`${prefs.user}:${prefs.token}`).toString("base64"),
+  Authorization: `Bearer ${prefs.token}`,
```

Comment:
If this works this way and user is not needed, we should take it out from being required

Reply:
I removed user from settings and from usage